acceptance-tests/cloudsql_pg_migration_test.go

```diff
@@ -150,7 +150,7 @@ var _ = Describe("Postgres service instance migration", Label("postgresql-data-m
 
 		By("performing the following actions against the target database:")
 		By("reading the data")
-		got := targetApp.GET("%s/%s", schema, key)
+		got := targetApp.GET("%s/%s", schema, key).String()
```

Comment:
non-blocking: Just wondering if it is worth it having a GetString method instead.

Reply:
I think this is more Go-like because it implements the `Stringer` interface: https://go.dev/tour/methods/17